changes-set.txt

```diff
@@ -86,6 +86,8 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+17-May-25 fimassd   [same]      Moved from GS's mathbox to main set.mm
```

Comment:
This was moved to main already before (see 6-May-25).

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4837/commits/3cb77ca77c03e28c1c877ddab4daed48dd91fb30